Marlin/src/lcd/extui/lib/mks_ui/draw_set.cpp

```diff
@@ -101,13 +98,8 @@ static void event_handler(lv_obj_t *obj, lv_event_t event) {
           }
           else {
             if (uiCfg.command_send == 1) {
-              buf[0] = 0xA5;
-              buf[1] = 0x07;
-              buf[2] = 0x00;
-              buf[3] = 0x00;
-              buf[4] = 0xFC;
-              raw_send_to_wifi(buf, 5);
-
+              uint8_t buf[] = { 0xA5, 0x07, 0x00, 0x00, 0xFC };
```

Comment:
What is this sequence of bytes. I suggest renaming `buf` to describe the contents.